tests/Generator/DiffGeneratorTest.php

```diff
@@ -102,7 +102,7 @@ public function testGenerate(): void
             ->with(self::logicalOr(
                 self::equalTo(['UPDATE table SET value = 2']),
                 self::equalTo(['UPDATE table SET value = 1']),
-            ), true, false, 80)
+            ), true, false, false, 80)
```

Comment:
Maybe we should start using named arguments, it's becoming hard to understand what's going on here.